### PR TITLE
[AspectJ] LPS 152106

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang,

This PR is to remove AspectJ concurrent usages. And the following also exists the use of AspectJ (test itself). Thanks!
`AspectJNewEnvJVMTestRuleTest`
`AspectJNewEnvClassLoaderTestRuleTest`
`AspectJNewEnvTestRule`

Janis.

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
ci:test:relevant

File: modules/apps/static/portal-configuration/portal-configuration-metatype-api/src/main/java/com/liferay/portal/configuration/metatype/bnd/util/ConfigurableUtil.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ private static <T> T _createConfigurableSnapshot(
 					interfaceClass, snapshotClassName);
 
 				try {
-					snapshotClass = (Class<T>)_defineClassMethod.invoke(
+					snapshotClass = (Class<T>)DefineClassMethodUtil.defineClass(
```
Comment (dantewang) [inline modules/apps/static/portal-configuration/portal-configuration-metatype-api/src/main/java/com/liferay/portal/configuration/metatype/bnd/util/ConfigurableUtil.java:83]: @JanisZhang Please don't modify `ConfigurableUtil` as it's the class being tested. Maybe you can subclass `ClassLoader`, write the `defineClass()` method, and replace the `_defineClassMethod` field in your test.

Comment (JanisZhang) [reply modules/apps/static/portal-configuration/portal-configuration-metatype-api/src/main/java/com/liferay/portal/configuration/metatype/bnd/util/ConfigurableUtil.java:83]: @dantewang noted, Thx

File: modules/apps/portal/portal-cluster-multiple/src/test/java/com/liferay/portal/cluster/multiple/internal/ClusterMasterExecutorImplTest.java [hunk 2]
```diff
@@ -634,13 +636,28 @@ public void testNotifyMasterTokenTransitionListeners() {
 				isMasterTokenReleasedNotified());
 	}
 
-	@Aspect
 	public static class ClusterExecutorAdvice {
```
Comment (dantewang) [inline modules/apps/portal/portal-cluster-multiple/src/test/java/com/liferay/portal/cluster/multiple/internal/ClusterMasterExecutorImplTest.java:639]: @JanisZhang Please avoid keeping a class called "advice" here as you've removed AspectJ usage.

**Pull request closed without merging**